index.bs

```diff
@@ -4068,13 +4220,579 @@ To <dfn>get the response data</dfn> given |response|:
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for network.continueRequest">
+The [=remote end steps=] given |session| and |command parameters| are:
+
+1. Let |blocked requests| be |session|'s [=blocked request map=].
+
+1. Let |request id| be |command parameters|["<code>request</code>"].
+
+1. If |blocked requests| does not [=map/contain=] |request id| then return
+   [=error=] with [=error code=] [=invalid argument=].
+
+   Issue: consider a "<code>no such request</code>" error.
+
+1. Let (|request|, |phase|, <var ignore>response</var>) be |blocked requests|[|request id|].
+
+1. If |phase| is not "<code>beforeRequestSent</code>", then return [=error=]
+   with [=error code=] [=invalid argument=].
+
+   Issue: consider a "<code>request already sent</code>" error.
+
+<!-- this has to be first because it's the only fallible operation-->
+
+1. If |command parameters| [=map/contains=] "<code>url</code>":
+
+  1. Let |url record| be the result of applying the [=URL parser=] to |command
+     parameters|["<code>url</code>"], with [=base URL=] null.
+
+  1. If |url record| is failure, return [=error=] with [=error code=] [=invalid
+     argument=].
+
+     TODO: Should we also resume here?
+
+  1. Let |request|'s [=request/url=] be |url record|.
+
+1. If|command parameters| [=map/contains=] "<code>method</code>":
+
+  1. Let |request|'s [=request/method=] be |command
+     parameters|["<code>method</code>"].
+
+1. If |command parameters| [=map/contains=] "<code>headers</code>":
+
+  1. Let |headers| be an empty [=/header list=].
+
+  1. For |header| in |command parameters|["<code>headers</code>"]:
+
+    1. Append [=deserialize header=] with |header| to |headers|.
+
+  1. Set |request|'s [=request/headers list=] to |headers|.
+
+1. If |command parameters| [=map/contains=] "<code>body</code>":
+
+  1. Let |body| be [=deserialize protocol bytes=] with |command
+     parameters|["<code>body</code>"].
+
+  1. Set |request|'s [=request/body=] to |body|.
+
+1. [=Resume=] with "<code>continue request</code>", |request id|, and (null,
+   "<code>incomplete</code>").
+
+1. Return [=success=] with [=data=] null.
+
+</div>
+
+#### The network.continueResponse Command ####  {#command-network-continueResponse}
+
+The <dfn export for=commands>network.continueResponse</dfn> command continues a
+response that's blocked by a [=network intercept=]. If called in the
+<code>beforeRequestSent</code> phase, this allows the request to be fulfilled
+without being sent. If called in the <code>responseStarted</code> phase, this
+allows the fetch response to be modified.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      network.ContinueResponse = {
+        method: "network.continueResponse",
+        params: network.ContinueResponseParameters
+      }
+
+      network.ContinueResponseParameters = {
```

Comment:
Does this mean it's not possible to modify the response body in this case? Yes, I know `network.provideResponse` exists, but not being able to modify that here seems an oversight.

Reply:
Resolved, via conversation in IRC. Will submit an issue once this PR lands in its initial form.